src/file.ts

```diff
@@ -3798,7 +3798,7 @@ class File extends ServiceObject<File> {
  * that a callback is omitted.
  */
 promisifyAll(File, {
-  exclude: ['request', 'setEncryptionKey'],
+  exclude: ['request', 'setEncryptionKey', 'publicUrl'],
```

Comment:
Please place `'publicUrl'` in alphabetic order.

Also please add `'publicUrl'` to the list to fix unit tests
https://github.com/googleapis/nodejs-storage/blob/642a9dff611be18b8b8ec6a14cfaed873daaf9f4/test/file.ts#L77